src/main/java/teammates/ui/controller/InstructorFeedbackPreviewAsInstructorAction.java

```diff
@@ -16,13 +16,13 @@ protected ActionResult execute() throws EntityDoesNotExistException {
 
         Assumption.assertNotNull(String.format(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE, 
                                                Const.ParamsNames.COURSE_ID), 
-                                 courseId);
+                                               courseId);
```

Comment:
They don't look aligned here on GitHub's UI but they are once it's opened in a text editor